Directory.Build.props

```diff
@@ -20,7 +20,7 @@
     <Platform Condition="'$(Platform)' == '' AND '$(BuildArchitecture)' == 'arm64'">$(BuildArchitecture)</Platform>
     <Platform Condition="'$(Platform)' == ''">x64</Platform>
 
-    <UseStableVersions Condition="'$(UseStableVersions)' == ''">false</UseStableVersions>
+    <UseStableVersions Condition="'$(UseStableVersions)' == ''">true</UseStableVersions>
```

Comment:
Can we delete this line? #1627

Reply:
No.  Without it, Roslyn wasn't building stable vesions.

Reply:
> Roslyn wasn't building stable versions.

Why is that a goal? The Microsoft Roslyn build didn't produce stable versions.
https://maestro-prod.westus2.cloudapp.azure.com/1297/https:%2F%2Fgithub.com%2Fdotnet%2Froslyn/68310/graph

> ![image](https://user-images.githubusercontent.com/12819531/98022525-582f6600-1dcb-11eb-8af4-e87baeee79e3.png)

And in Version.Details.xml, `<Dependency Name="Microsoft.NETCore.Compilers" Version="3.8.0-5.20519.18" ...`

vs.
```
  [14:25:41.87] Building 'roslyn'...done
  New NuGet package(s) after building roslyn:
    -> Microsoft.CodeAnalysis.Build.Tasks 3.8.0
    -> Microsoft.CodeAnalysis.Common 3.8.0
```

Reply:
Hmm...  A later repo was looking for a stable version and wasn't finding it.  I was mistaken in my fix for that.

Reply:
I think it's fine to leave https://github.com/dotnet/source-build/issues/1627 tracking it for followup but wanted to dig into any issues to make it easier to tackle later.

Reply:
Without the patch, the production build produces `Microsoft.NETCore.Compilers.3.8.0-dev.20519.18.nupkg`.  The "dev" version doesn't seem right.

Reply:
That sounds like a missing `-ci` or `OfficialBuildId` or something along those lines.

Reply:
I didn't change roslyn.proj.  It has `OfficialBuild=true` here: https://github.com/dotnet/source-build/blob/92d2aefc755e365d09f661fbaf9e0c3755f2179d/repos/roslyn.proj#L12

Reply:
Adding `--ci` to roslyn.proj caused it to generate the desired version.  `Microsoft.Net.Compilers.Toolset.3.8.0-5.20519.18.nupkg`  I'll fix this once this PR is merged under #1627